api/allocations_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"time"
 
-	"github.com/hashicorp/nomad/helper/uuid"
```

Comment:
In a follow-up PR, I'll nuke `helper/uuid` for `hashicorp/go-uuid`?  Given that we already depend on go-uuid (as showed by vendor file), it doesn't make sense to have our own helper package.